enterprise/management/commands/unlink_enterprise_customer_learners.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+Django management command for unlink the learners and removing DSC.
+"""
+import csv
+import logging
+
+from django.core.management import BaseCommand
+
+from edx_django_utils.cache import get_cache_key, TieredCache
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerUser, \
+    PendingEnterpriseCustomerUser
+from django.contrib import auth
+
+from enterprise.utils import get_enterprise_customer_user
+
+User = auth.get_user_model()  # pylint: disable=invalid-name
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django management command for creating DataSharingConsent records
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--data-csv',
+            action='store',
+            dest='data_csv',
+            help='Path of csv to read enterprise learner data',
+            type=str,
+        )
+
+        parser.add_argument(
+            '-e',
+            '--enterprise_customer_uuid',
+            action='store',
+            dest='enterprise_customer_uuid',
+            help='Run this command for only the given EnterpriseCustomer UUID.'
+        )
+
+    def handle(self, *args, **options):
+        csv_path = options['data_csv']
+        enterprise_customer_uuid = options['enterprise_customer_uuid']
+        enterprise_customer = EnterpriseCustomer.objects.get(uuid=enterprise_customer_uuid)
+
+        LOGGER.info('Started Unlinking and removing DSC')
+
+        results = {}
+
+        with open(csv_path) as csv_file:
```

Comment:
For a one-time thing this is probably fine, but I would recommend reading the file first and then doing the operations outside of the `with open` so you don't keep the file handler open longer than you need to.